<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.crm.qa.testcases.LoginPageTest" tests="6" time="33.838" ignored="0" failures="1" timestamp="06 Dec 2017 15:19:54 GMT" skipped="3" hostname="Sumits-iMac.local" errors="0">
  <testcase name="loginPageTitleTest" time="4.306" classname="com.crm.qa.testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="loginTest" time="12.810" classname="com.crm.qa.testcases.LoginPageTest"/>
  <testcase name="loginPageTitleTest" time="3.457" classname="com.crm.qa.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [1 Free CRM for Any Business: Online Customer Relationship Software] but found [#1 Free CRM for Any Business: Online Customer Relationship Software]">
      <![CDATA[java.lang.AssertionError: expected [1 Free CRM for Any Business: Online Customer Relationship Software] but found [#1 Free CRM for Any Business: Online Customer Relationship Software]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.crm.qa.testcases.LoginPageTest.loginPageTitleTest(LoginPageTest.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:998)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="loginPageTitleTest" time="4.331" classname="com.crm.qa.testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="crmLogoImageTest" time="4.562" classname="com.crm.qa.testcases.LoginPageTest"/>
  <testcase name="loginPageTitleTest" time="4.372" classname="com.crm.qa.testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
</testsuite> <!-- com.crm.qa.testcases.LoginPageTest -->
